### PR TITLE
build: Fix `make apk` if ccache enabled, and enable it on CI

### PR DESCRIPTION
**Pull request by hebasto**
On master (456c8d6cd80fc3461957a3553a0483756396b988) `make apk` is broken for Android targets if the build system was configured with `--enable-ccache` explicitly or by default.

This PR fixes this bug, and enables `ccache` for "ARM64 Android APK" Cirrus CI job.

### CODE REVIEW

**Comment by hebasto** [comment]
cc @icota

File: configure.ac [hunk 1]
```diff
@@ -1622,6 +1622,8 @@ AC_MSG_RESULT($build_bitcoin_libs)
 
 AC_LANG_POP
 
+CC_UNPREFIXED=$CC
+AC_SUBST(CC_UNPREFIXED)
```
Comment (icota) [inline configure.ac:1626]: Why is this needed?

Comment (hebasto) [reply configure.ac:1626]: To enable `ccache` the code below in `configure.ac` makes the `CC` variable looks like that
```
<path_to_ccache> <path_to_compiler>
```
Then https://github.com/bitcoin/bitcoin/blob/5c041cb348e667672c8b5ed504dba05adf60783b/src/Makefile.qt.include#L390

the `make` function `$(dir $(CC))` treats the `CC` variable as a list which consist of two paths, and, eventually, the `cp` command fails.

**Review by icota — APPROVED** [review]

**Comment by icota** [comment]
tACK https://github.com/bitcoin/bitcoin/pull/22076/commits/865627be85778799f86122653253c5448b5ef41a

File: configure.ac [hunk 2]
```diff
@@ -1622,6 +1622,8 @@ AC_MSG_RESULT($build_bitcoin_libs)
 
 AC_LANG_POP
 
+CC_UNPREFIXED=$CC
```
Comment (fanquake) [inline configure.ac:1625]: This could really use some sort of comment.

**Comment by hebasto** [comment]
Updated 865627be85778799f86122653253c5448b5ef41a -> db1bb18b87de92068689dbd4aa0c4c4f500ed9ab ([pr22076.01](https://github.com/hebasto/bitcoin/commits/pr22076.01) -> [pr22076.02](https://github.com/hebasto/bitcoin/commits/pr22076.02), [diff](https://github.com/hebasto/bitcoin/compare/pr22076.01..pr22076.02)):

- addressed @fanquake's [comment](https://github.com/bitcoin/bitcoin/pull/22076#discussion_r642270347)

File: configure.ac (hunk 2 above)
Comment (hebasto) [reply configure.ac:1628]: Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/22076#issuecomment-853202108).

**Comment by hebasto** [comment]
@icota @fanquake

Do you mind having another look into this pr?

**Comment by BlockMechanic** [comment]
This PR is life !

**Comment by icota** [comment]
@BlockMechanic Maybe even larger than life :earth_africa: 

@hebasto I am running into this:
```
make[1]: Leaving directory '/home/igor/Code/bitcoin/src'
make -C .. bitcoin_qt_apk
make[1]: Entering directory '/home/igor/Code/bitcoin/src'
tar: ../depends/sources/qtbase-everywhere-src-5.12.11.tar.xz: Not found in archive
tar: Exiting with failure status due to previous errors
tar: ../depends/sources/qtbase-everywhere-src-5.12.11.tar.xz: Not found in archive
tar: Exiting with failure status due to previous errors
tar: ../depends/sources/qtbase-everywhere-src-5.12.11.tar.xz: Not found in archive
tar: Exiting with failure status due to previous errors
mkdir -p qt/android/libs/arm64-v8a
cp /home/igor/Code/bitcoin/depends/SDKs/android/ndk/21.1.6352462/toolchains/llvm/prebuilt/linux-x86_64/bin//../sysroot/usr/lib/aarch64-linux-android/libc++_shared.so qt/android/libs/arm64-v8a
tar xf ../depends/sources/qtbase-everywhere-src-5.12.10.tar.xz ../depends/sources/qtbase-everywhere-src-5.12.11.tar.xz -C qt/android/src/ src/android/jar/src --strip-components=5
tar: ../depends/sources/qtbase-everywhere-src-5.12.11.tar.xz: Not found in archive
tar: src/android/jar/src: Not found in archive
tar: Exiting with failure status due to previous errors
make[1]: *** [Makefile:18020: bitcoin_qt_apk] Error 2
make[1]: Leaving directory '/home/igor/Code/bitcoin/src'
make: *** [Makefile:11: apk] Error 2
```

**Comment by hebasto** [comment]
@icota 

> @hebasto I am running into this:

And your previous steps?

**Comment by icota** [comment]
`FILE_ENV="./ci/test/00_setup_env_android.sh" ./ci/test_run_all.sh`

**Comment by BlockMechanic** [comment]
@icota i got around that by changing 
https://github.com/bitcoin/bitcoin/blob/master/src/Makefile.qt.include#L389

to 

`QT_BASE_PATH = $(shell find $(top_srcdir)/depends/sources/ -maxdepth 1 -type f -regex ".*qtbase.*\.tar.xz")
`

**Comment by hebasto** [comment]
@icota @BlockMechanic 

Do changes from #22555 fix an issue with `QT_BASE_PATH` here for you?

**Comment by icota** [comment]
> Do changes from #22555 fix an issue with `QT_BASE_PATH` here for you?

They do. Is #22555 meant to be a replacement?

**Comment by hebasto** [comment]
@icota
> Is #22555 meant to be a replacement?

If you mean a replacement for this PR, it is not.

**Comment by jarolrod** [comment]
GUIX hashes:

```
find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum

4130e0445ffb930668c50b2e05ef4e3c883f2706c99c76094092cf073242a593  guix-build-db1bb18b87de/output/aarch64-linux-gnu/bitcoin-db1bb18b87de-aarch64-linux-gnu-debug.tar.gz
e018eaba415f2b81154cf4ae7f116517b2b3da5c6e060a6bb69f85601bf3a72b  guix-build-db1bb18b87de/output/aarch64-linux-gnu/bitcoin-db1bb18b87de-aarch64-linux-gnu.tar.gz
1ad2b249a675eb2c41b725ca763232713758d8e15f78f815b67bb7f31162a398  guix-build-db1bb18b87de/output/aarch64-linux-gnu/inputs.SHA256SUMS
c160467e62a90a6c5ee6276fe22484fe1542c51f512552efaff359efd3383f76  guix-build-db1bb18b87de/output/arm-linux-gnueabihf/bitcoin-db1bb18b87de-arm-linux-gnueabihf-debug.tar.gz
270bafb676d2346afa5d12d1f800f901ed225f3efe26a7d44247643c56b7feb1  guix-build-db1bb18b87de/output/arm-linux-gnueabihf/bitcoin-db1bb18b87de-arm-linux-gnueabihf.tar.gz
1ad2b249a675eb2c41b725ca763232713758d8e15f78f815b67bb7f31162a398  guix-build-db1bb18b87de/output/arm-linux-gnueabihf/inputs.SHA256SUMS
e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855  guix-build-db1bb18b87de/output/dist-archive/SKIPATTEST.TAG
a5ccc1fdd1713b9205d4e1319a70f69c5f1fd9c9865fff95c97f34af56bb89c2  guix-build-db1bb18b87de/output/dist-archive/bitcoin-db1bb18b87de.tar.gz
2a8648500d9c49f27113312484a96c66f2ea11f82f75a93b3e71c9f7d305860c  guix-build-db1bb18b87de/output/powerpc64-linux-gnu/bitcoin-db1bb18b87de-powerpc64-linux-gnu-debug.tar.gz
8e962332c762cd6afa665aad5b9980b5813b555bd46565bc5cc9594e3ad74d86  guix-build-db1bb18b87de/output/powerpc64-linux-gnu/bitcoin-db1bb18b87de-powerpc64-linux-gnu.tar.gz
1ad2b249a675eb2c41b725ca763232713758d8e15f78f815b67bb7f31162a398  guix-build-db1bb18b87de/output/powerpc64-linux-gnu/inputs.SHA256SUMS
7d6bb9176a8c3cfc366a83efac6201e182d6880431c42b6737de05d3dc745f4b  guix-build-db1bb18b87de/output/powerpc64le-linux-gnu/bitcoin-db1bb18b87de-powerpc64le-linux-gnu-debug.tar.gz
96ca62e533628c004ce1981b8d809b04dd0c0834a30e03d5c17fbdce1ae1ec08  guix-build-db1bb18b87de/output/powerpc64le-linux-gnu/bitcoin-db1bb18b87de-powerpc64le-linux-gnu.tar.gz
1ad2b249a675eb2c41b725ca763232713758d8e15f78f815b67bb7f31162a398  guix-build-db1bb18b87de/output/powerpc64le-linux-gnu/inputs.SHA256SUMS
b0a8cc447ae7ff6c7669d5aaa6a3ff85e959d8890dc9587034dea63c61d8bfa2  guix-build-db1bb18b87de/output/riscv64-linux-gnu/bitcoin-db1bb18b87de-riscv64-linux-gnu-debug.tar.gz
a8b4e8c5be2bb6e9722170f9dc4e7bd29e0643f83e039667f9b45a92706423ad  guix-build-db1bb18b87de/output/riscv64-linux-gnu/bitcoin-db1bb18b87de-riscv64-linux-gnu.tar.gz
1ad2b249a675eb2c41b725ca763232713758d8e15f78f815b67bb7f31162a398  guix-build-db1bb18b87de/output/riscv64-linux-gnu/inputs.SHA256SUMS
0f5a546f5ebe1823ff66212e558eabd252908ebc880d4c9fc39bfb1cb156fab4  guix-build-db1bb18b87de/output/x86_64-apple-darwin18/bitcoin-db1bb18b87de-osx-unsigned.dmg
2991f7b02501d0e45343861a9895f1c0e6da83455c246d1d4bb020a8850a946f  guix-build-db1bb18b87de/output/x86_64-apple-darwin18/bitcoin-db1bb18b87de-osx-unsigned.tar.gz
ca392b05c8124bb9cbd896113b8a17790512189b128b151b5b9a249703e126a5  guix-build-db1bb18b87de/output/x86_64-apple-darwin18/bitcoin-db1bb18b87de-osx64.tar.gz
1ad2b249a675eb2c41b725ca763232713758d8e15f78f815b67bb7f31162a398  guix-build-db1bb18b87de/output/x86_64-apple-darwin18/inputs.SHA256SUMS
7e19e4b76d2c31c0d10d62746eddc26c9ee09d310befd7e6e6fdc153e8e65d1e  guix-build-db1bb18b87de/output/x86_64-linux-gnu/bitcoin-db1bb18b87de-x86_64-linux-gnu-debug.tar.gz
ac7f49ca8a54a4419d37f846272f48dc44deb05d811efa219288d14b3d9c0d05  guix-build-db1bb18b87de/output/x86_64-linux-gnu/bitcoin-db1bb18b87de-x86_64-linux-gnu.tar.gz
1ad2b249a675eb2c41b725ca763232713758d8e15f78f815b67bb7f31162a398  guix-build-db1bb18b87de/output/x86_64-linux-gnu/inputs.SHA256SUMS
a8c1e6a733cf3275a7462792bdd70d7c1c5c877ad3e1c10fb7569956171cce30  guix-build-db1bb18b87de/output/x86_64-w64-mingw32/bitcoin-db1bb18b87de-win-unsigned.tar.gz
4ea3afc292e6b554a46570c36bcbe40cc427c43ef1e844b81d5ca85b1f57b98f  guix-build-db1bb18b87de/output/x86_64-w64-mingw32/bitcoin-db1bb18b87de-win64-debug.zip
a0e2d4d7e63fcaf666cc1c61a2857180f9f8d994b5b65c102121c0ca859325b0  guix-build-db1bb18b87de/output/x86_64-w64-mingw32/bitcoin-db1bb18b87de-win64-setup-unsigned.exe
b728297163b0d7bc5a86a0a30f9508d2dbdac715a4bb72a554a09ee3985525b0  guix-build-db1bb18b87de/output/x86_64-w64-mingw32/bitcoin-db1bb18b87de-win64.zip
1ad2b249a675eb2c41b725ca763232713758d8e15f78f815b67bb7f31162a398  guix-build-db1bb18b87de/output/x86_64-w64-mingw32/inputs.SHA256SUMS
```

**Comment by hebasto** [comment]
#### Guix builds:
```
$ find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
4130e0445ffb930668c50b2e05ef4e3c883f2706c99c76094092cf073242a593  guix-build-db1bb18b87de/output/aarch64-linux-gnu/bitcoin-db1bb18b87de-aarch64-linux-gnu-debug.tar.gz
e018eaba415f2b81154cf4ae7f116517b2b3da5c6e060a6bb69f85601bf3a72b  guix-build-db1bb18b87de/output/aarch64-linux-gnu/bitcoin-db1bb18b87de-aarch64-linux-gnu.tar.gz
1ad2b249a675eb2c41b725ca763232713758d8e15f78f815b67bb7f31162a398  guix-build-db1bb18b87de/output/aarch64-linux-gnu/inputs.SHA256SUMS
c160467e62a90a6c5ee6276fe22484fe1542c51f512552efaff359efd3383f76  guix-build-db1bb18b87de/output/arm-linux-gnueabihf/bitcoin-db1bb18b87de-arm-linux-gnueabihf-debug.tar.gz
270bafb676d2346afa5d12d1f800f901ed225f3efe26a7d44247643c56b7feb1  guix-build-db1bb18b87de/output/arm-linux-gnueabihf/bitcoin-db1bb18b87de-arm-linux-gnueabihf.tar.gz
1ad2b249a675eb2c41b725ca763232713758d8e15f78f815b67bb7f31162a398  guix-build-db1bb18b87de/output/arm-linux-gnueabihf/inputs.SHA256SUMS
e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855  guix-build-db1bb18b87de/output/dist-archive/SKIPATTEST.TAG
a5ccc1fdd1713b9205d4e1319a70f69c5f1fd9c9865fff95c97f34af56bb89c2  guix-build-db1bb18b87de/output/dist-archive/bitcoin-db1bb18b87de.tar.gz
2a8648500d9c49f27113312484a96c66f2ea11f82f75a93b3e71c9f7d305860c  guix-build-db1bb18b87de/output/powerpc64-linux-gnu/bitcoin-db1bb18b87de-powerpc64-linux-gnu-debug.tar.gz
8e962332c762cd6afa665aad5b9980b5813b555bd46565bc5cc9594e3ad74d86  guix-build-db1bb18b87de/output/powerpc64-linux-gnu/bitcoin-db1bb18b87de-powerpc64-linux-gnu.tar.gz
1ad2b249a675eb2c41b725ca763232713758d8e15f78f815b67bb7f31162a398  guix-build-db1bb18b87de/output/powerpc64-linux-gnu/inputs.SHA256SUMS
7d6bb9176a8c3cfc366a83efac6201e182d6880431c42b6737de05d3dc745f4b  guix-build-db1bb18b87de/output/powerpc64le-linux-gnu/bitcoin-db1bb18b87de-powerpc64le-linux-gnu-debug.tar.gz
96ca62e533628c004ce1981b8d809b04dd0c0834a30e03d5c17fbdce1ae1ec08  guix-build-db1bb18b87de/output/powerpc64le-linux-gnu/bitcoin-db1bb18b87de-powerpc64le-linux-gnu.tar.gz
1ad2b249a675eb2c41b725ca763232713758d8e15f78f815b67bb7f31162a398  guix-build-db1bb18b87de/output/powerpc64le-linux-gnu/inputs.SHA256SUMS
b0a8cc447ae7ff6c7669d5aaa6a3ff85e959d8890dc9587034dea63c61d8bfa2  guix-build-db1bb18b87de/output/riscv64-linux-gnu/bitcoin-db1bb18b87de-riscv64-linux-gnu-debug.tar.gz
a8b4e8c5be2bb6e9722170f9dc4e7bd29e0643f83e039667f9b45a92706423ad  guix-build-db1bb18b87de/output/riscv64-linux-gnu/bitcoin-db1bb18b87de-riscv64-linux-gnu.tar.gz
1ad2b249a675eb2c41b725ca763232713758d8e15f78f815b67bb7f31162a398  guix-build-db1bb18b87de/output/riscv64-linux-gnu/inputs.SHA256SUMS
0f5a546f5ebe1823ff66212e558eabd252908ebc880d4c9fc39bfb1cb156fab4  guix-build-db1bb18b87de/output/x86_64-apple-darwin18/bitcoin-db1bb18b87de-osx-unsigned.dmg
2991f7b02501d0e45343861a9895f1c0e6da83455c246d1d4bb020a8850a946f  guix-build-db1bb18b87de/output/x86_64-apple-darwin18/bitcoin-db1bb18b87de-osx-unsigned.tar.gz
ca392b05c8124bb9cbd896113b8a17790512189b128b151b5b9a249703e126a5  guix-build-db1bb18b87de/output/x86_64-apple-darwin18/bitcoin-db1bb18b87de-osx64.tar.gz
1ad2b249a675eb2c41b725ca763232713758d8e15f78f815b67bb7f31162a398  guix-build-db1bb18b87de/output/x86_64-apple-darwin18/inputs.SHA256SUMS
7e19e4b76d2c31c0d10d62746eddc26c9ee09d310befd7e6e6fdc153e8e65d1e  guix-build-db1bb18b87de/output/x86_64-linux-gnu/bitcoin-db1bb18b87de-x86_64-linux-gnu-debug.tar.gz
ac7f49ca8a54a4419d37f846272f48dc44deb05d811efa219288d14b3d9c0d05  guix-build-db1bb18b87de/output/x86_64-linux-gnu/bitcoin-db1bb18b87de-x86_64-linux-gnu.tar.gz
1ad2b249a675eb2c41b725ca763232713758d8e15f78f815b67bb7f31162a398  guix-build-db1bb18b87de/output/x86_64-linux-gnu/inputs.SHA256SUMS
a8c1e6a733cf3275a7462792bdd70d7c1c5c877ad3e1c10fb7569956171cce30  guix-build-db1bb18b87de/output/x86_64-w64-mingw32/bitcoin-db1bb18b87de-win-unsigned.tar.gz
4ea3afc292e6b554a46570c36bcbe40cc427c43ef1e844b81d5ca85b1f57b98f  guix-build-db1bb18b87de/output/x86_64-w64-mingw32/bitcoin-db1bb18b87de-win64-debug.zip
a0e2d4d7e63fcaf666cc1c61a2857180f9f8d994b5b65c102121c0ca859325b0  guix-build-db1bb18b87de/output/x86_64-w64-mingw32/bitcoin-db1bb18b87de-win64-setup-unsigned.exe
b728297163b0d7bc5a86a0a30f9508d2dbdac715a4bb72a554a09ee3985525b0  guix-build-db1bb18b87de/output/x86_64-w64-mingw32/bitcoin-db1bb18b87de-win64.zip
1ad2b249a675eb2c41b725ca763232713758d8e15f78f815b67bb7f31162a398  guix-build-db1bb18b87de/output/x86_64-w64-mingw32/inputs.SHA256SUMS
```

**Comment by hebasto** [comment]
Rebased db1bb18b87de92068689dbd4aa0c4c4f500ed9ab -> 1de0b494b6e645956753e71731dba4dbc86215e6 ([pr22076.02](https://github.com/hebasto/bitcoin/commits/pr22076.02) -> [pr22076.03](https://github.com/hebasto/bitcoin/commits/pr22076.03)) due to the conflict with #23286.

Also, on my system `FILE_ENV="./ci/test/00_setup_env_android.sh" ./ci/test_run_all.sh` passed.

@icota @BlockMechanic 
Mind having another round of reviewing please?

**Comment by jarolrod** [comment]
Contributing GUIX hashes, mine match @hebasto second round of hashes:

```
9fe282ae230b26d6eb50524a613d22e727ec83899f2db4f220e2e9a394cbebfa  guix-build-1de0b494b6e6/output/aarch64-linux-gnu/SHA256SUMS.part
01b86de3c6c5bc4bb6c6fd3482d88429bff6327367d590e730ed6a0511e1eb76  guix-build-1de0b494b6e6/output/aarch64-linux-gnu/bitcoin-1de0b494b6e6-aarch64-linux-gnu-debug.tar.gz
7022d07c209cdf904457c4a9cacceb7db58ac3855a614bceaf14bea61d110a97  guix-build-1de0b494b6e6/output/aarch64-linux-gnu/bitcoin-1de0b494b6e6-aarch64-linux-gnu.tar.gz
8ee326f6ff347edc0b2132d53a23b5b315a8365fd353afa95d90968d3601c981  guix-build-1de0b494b6e6/output/arm-linux-gnueabihf/SHA256SUMS.part
0847523cfb8cccb2364bed5f17d3c085a77f604e3fb3a5bf7d464d93327be02a  guix-build-1de0b494b6e6/output/arm-linux-gnueabihf/bitcoin-1de0b494b6e6-arm-linux-gnueabihf-debug.tar.gz
a825b0588069a9cbbf3b952cd65392e4b712811053c548e4cda7c06b30a6bfd2  guix-build-1de0b494b6e6/output/arm-linux-gnueabihf/bitcoin-1de0b494b6e6-arm-linux-gnueabihf.tar.gz
b046611c1054c2bf935570a8554108d4781f404dd865380968895d2114afa6ff  guix-build-1de0b494b6e6/output/dist-archive/bitcoin-1de0b494b6e6.tar.gz
01865d6e2a69506becb466dbb1cac454860d305a032ee4e8f51921653423063a  guix-build-1de0b494b6e6/output/powerpc64-linux-gnu/SHA256SUMS.part
83e41d320024af672aaa4f5128e2b747dec7db4ea4e553e0c8c5b9915343c566  guix-build-1de0b494b6e6/output/powerpc64-linux-gnu/bitcoin-1de0b494b6e6-powerpc64-linux-gnu-debug.tar.gz
b07df6652b8503aaa63931200b04b6b0f21049927a1c1fb730d9f8fc8c2efce9  guix-build-1de0b494b6e6/output/powerpc64-linux-gnu/bitcoin-1de0b494b6e6-powerpc64-linux-gnu.tar.gz
0156d57e63132d4eb1ca5cbbbf2f066d170749ac737184b7f8bf009483569000  guix-build-1de0b494b6e6/output/powerpc64le-linux-gnu/SHA256SUMS.part
2747836562c7b1009c7f72026614398ebb3a50f5eab213f44433d72c96e889a8  guix-build-1de0b494b6e6/output/powerpc64le-linux-gnu/bitcoin-1de0b494b6e6-powerpc64le-linux-gnu-debug.tar.gz
7a26a075b21151f8e9b3c03042ef336276a5690cc3596a46198fd2d3925768be  guix-build-1de0b494b6e6/output/powerpc64le-linux-gnu/bitcoin-1de0b494b6e6-powerpc64le-linux-gnu.tar.gz
95ad807e17343cdd52c72e1090cf4e36d2646fa7931e483120b3cacb830399bc  guix-build-1de0b494b6e6/output/riscv64-linux-gnu/SHA256SUMS.part
9af34c87cfd57fa8239331430b3fcc9711998f47e966c16046b37cc273a01223  guix-build-1de0b494b6e6/output/riscv64-linux-gnu/bitcoin-1de0b494b6e6-riscv64-linux-gnu-debug.tar.gz
e02d1fed0f9de626bbb68020357562b0a3cc3b492ffae779d91f8d43f7bc7f59  guix-build-1de0b494b6e6/output/riscv64-linux-gnu/bitcoin-1de0b494b6e6-riscv64-linux-gnu.tar.gz
a75ce7e0cef5beb2d0230f9085499ad92858a024a6140fb962be966f864affa7  guix-build-1de0b494b6e6/output/x86_64-apple-darwin19/SHA256SUMS.part
0829cef5163ff8420ef17ee41699e1be2f6b708ea63eb7dfc275fab055af5808  guix-build-1de0b494b6e6/output/x86_64-apple-darwin19/bitcoin-1de0b494b6e6-osx-unsigned.dmg
1b59fcc17f385e0b21496cfefc222f78850f5f3138f403124c89c1b6cb2ad13d  guix-build-1de0b494b6e6/output/x86_64-apple-darwin19/bitcoin-1de0b494b6e6-osx-unsigned.tar.gz
1fcc2dbfdba681d81c35c3628d8e89da29d0f2ba7f8050274c6f1177e6378a05  guix-build-1de0b494b6e6/output/x86_64-apple-darwin19/bitcoin-1de0b494b6e6-osx64.tar.gz
3ade4806716714ab402e71ba22060309aafe2643811e3973b1609db582aa8439  guix-build-1de0b494b6e6/output/x86_64-linux-gnu/SHA256SUMS.part
5f1f79a3e19d179e6bc9e5e73aff88412b522cb7feb4691e18f525092b8a4df1  guix-build-1de0b494b6e6/output/x86_64-linux-gnu/bitcoin-1de0b494b6e6-x86_64-linux-gnu-debug.tar.gz
4d33d5bbabf5682206f77b1a7e676dcd4c896923616425e15bd28ad0a265fc59  guix-build-1de0b494b6e6/output/x86_64-linux-gnu/bitcoin-1de0b494b6e6-x86_64-linux-gnu.tar.gz
bd8b8a496b0243f25a7cb396812a07262bc13ec9cb8d1054fb399de75fa53c23  guix-build-1de0b494b6e6/output/x86_64-w64-mingw32/SHA256SUMS.part
34c1dd41491870261c9323e5e7f3a63003215149d175742f8f030ed75821ac1b  guix-build-1de0b494b6e6/output/x86_64-w64-mingw32/bitcoin-1de0b494b6e6-win-unsigned.tar.gz
33af8329bb223003ee6658ac523c66e65b484e8e09b0cada206c8711cd8d2b1f  guix-build-1de0b494b6e6/output/x86_64-w64-mingw32/bitcoin-1de0b494b6e6-win64-debug.zip
5a89fbd7402a6d9f1cd088051c708b7a3957202e4416c2491f03b04c7f0cab39  guix-build-1de0b494b6e6/output/x86_64-w64-mingw32/bitcoin-1de0b494b6e6-win64-setup-unsigned.exe
c77048890ed8d4591ef1161f82115289418181c7d9108a1d4ec532d9356c37af  guix-build-1de0b494b6e6/output/x86_64-w64-mingw32/bitcoin-1de0b494b6e6-win64.zip
```

**Review by fanquake — CHANGES_REQUESTED** [review]
You can use my suggestion and drop the changes from `configure.ac`.

File: src/Makefile.qt.include [hunk 3]
```diff
@@ -391,7 +391,7 @@ QT_BASE_TLD = $(shell tar tf $(QT_BASE_PATH) --exclude='*/*')
 
 bitcoin_qt_apk: FORCE
 	mkdir -p $(APK_LIB_DIR)
-	cp $(dir $(CC))../sysroot/usr/lib/$(host_alias)/libc++_shared.so $(APK_LIB_DIR)
+	cp $(dir $(CC_UNPREFIXED))../sysroot/usr/lib/$(host_alias)/libc++_shared.so $(APK_LIB_DIR)
```
[the remark this answers is not in the document]
Comment (hebasto) [reply src/Makefile.qt.include:394]: Is it guaranteed that this approach will work for all cases of user-provided `CC`?

**Comment by hebasto** [comment]
Updated 1de0b494b6e645956753e71731dba4dbc86215e6 -> 15fb57556ee9a5f7332c9fff61edc0cc67634053 ([pr22076.03](https://github.com/hebasto/bitcoin/commits/pr22076.03) -> [pr22076.04](https://github.com/hebasto/bitcoin/commits/pr22076.04)):

- addressed @fanquake's [comment](https://github.com/bitcoin/bitcoin/pull/22076#discussion_r744339610)

File: src/Makefile.qt.include (hunk 3 above)
[the remark this answers is not in the document]
Comment (hebasto) [reply src/Makefile.qt.include]: Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/22076#issuecomment-962824073).

**Review by fanquake — APPROVED** [review]
ACK 15fb57556ee9a5f7332c9fff61edc0cc67634053
